### ISSUE TITLE
Flame figures on skin biopsy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9989]
**Preferred term label:**
Flame figures on skin biopsy

**Synonyms**
Central part consisting of collagen fibers and eosinophilic granules, surrounded by a histiocytic and eosinophilic infiltrate

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Wells syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000090
- label: Dermal flame figures
- parents: [Abnormal dermis morphology (HP:0034924)](https://hpo.jax.org/app/browse/term/HP:0034924)
- pmid: PMID:30335327
- def: Flame-figures are composed of a central part consisting of collagen fibers and eosinophilic granules, surrounded by a histiocytic and eosinophilic infiltrate. Subsequently, eosinophils tend to disappear and are replaced by phagocytic granulomas, consisting of histiocytes and sometimes giant cells, around the flame figures.
- comment: Flame figures may be observed in Wells syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9989
- synonyms:
